CMakeLists.txt

```diff
@@ -36,7 +36,7 @@ set(MKL_THREADING "tbb_thread")
 set(MKL_VERSION_2024 FALSE)
 find_package(MKL QUIET)
 if(MKL_FOUND)
-    if(TARGET MKL::MKL_SYCL)
+    if(MKL_VERSION GREATER_EQUAL 2024.0.0)
```

Comment:
Did you try

```
if (MKL_VERSION VERSION_GREATER_EQUAL "2024.0.0")
```

Reply:
@oleksandr-pavlyk
I just tested this and it works too.
Should I change checks?

Reply:
Yes, please

Reply:
done